cmd/cmdutil/helpers.go

```diff
@@ -45,6 +46,16 @@ func AddLimitFlag(cmd *cobra.Command, limit int, desc string) {
 	cmd.Flags().IntP("limit", "l", limit, desc)
 }
 
+// AddMaxConcurrencyFlag adds --max-concurrency to command
+func AddMaxConcurrencyFlag(cmd *cobra.Command, defaultMax string, desc string) {
```

Comment:
These will need to be `cmd.Flags().String` instead of `cmd.PersistentFlags().String` since they're added to a subcommand instead of the parent command.